extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -65,7 +65,8 @@ private static function renderImage( $media, $id ) {
 					'title' => $media['title'],
 					'fileUrl' => $media['fileUrl'],
 					'caption' => $media['caption'] ?? '',
-					'link' => $media['link'] ?? '',
+					'href' => $media['href'],
+					'isLinkedByUser' => $media['isLinkedByUser'],
```

Comment:
The issue is that now all the thumbnails are technically speaking links. That's why I wanted to explicitly state that user added a link and not us. I'm not a huge fan of `isLinkedByUser` neither but I don't have a better name that would tell the full story.